dev/io.openliberty.jakarta.mail.2.0/src/io/openliberty/jakarta/mail/MailSessionRegistrar.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.jakarta.mail;
```

Comment:
As noted above, I think this package name should be `io.openliberty.mail` because this is not a jakarta API